sunpy/time/time.py

```diff
@@ -354,6 +354,7 @@ def is_time_in_given_format(time_string, time_format):
         return False
 
 
+@deprecated("2.0", alternative='(time.jd - 2415020) / 36525')
```

Comment:
The alternative here is knowing these magic numbers? Isn't there a way to do it with `Time`?

Reply:
Dunno/I don't even know where the magic numbers are documented or officially defined elsewhere.

Reply:
I believe it traces back to [Newcomb's _Tables of the Sun_](https://en.wikipedia.org/wiki/Newcomb%27s_Tables_of_the_Sun).  A rendition that makes better use of `Time` would be `(time - Time('J1900.0', scale='tt')) / (36525*u.day)` (and I guess a `.value` for the whole thing for just the number).

Reply:
There doesn't appear to be a pre-defined unit in Astropy for Julian century.

Reply:
I realise I am re-litigating this somewhat, but this seems like it's non-trivial to do without this function, so why did it get removed?

Reply:
I removed it probably.

Reply:
I'm fine with keeping it, but the scale bug needs to be fixed and the docstring ought to be improved.